tokens/src/lib.rs

```diff
@@ -171,7 +172,7 @@ pub mod module {
 			+ MaybeSerializeDeserialize;
 
 		/// The currency ID type
-		type CurrencyId: Parameter + Member + Copy + MaybeSerializeDeserialize + Ord;
+		type CurrencyId: Parameter + Member + Copy + MaybeSerializeDeserialize + Ord + Default;
```

Comment:
why `Default` is required?

Reply:
https://github.com/paritytech/substrate/blob/master/frame/support/src/traits/tokens/misc.rs#L164
AssetId need `Default`